minerl/Malmo/Minecraft/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
```

Comment:
is this backslash important? I have it in my gradle-wrapper.properties, based on your slack message https://openai.slack.com/archives/C01RJA7BEPP/p1616105094042400

Reply:
I'm going to go with this is not important unless it breaks something somewhere =) which it does not seem to do for ci